src/app/import/components/helpers/upload-specific.ts

```diff
@@ -26,6 +26,7 @@ export class RecordNewUploadHelper {
 
 export class RecordAddUploadHelper {
     projectName: string = '';
+    hasNotGdprEmbeddings: boolean = false;
```

Comment:
double negative is hard to read so maybe a flip can help 
```suggestion
    hasGdprEmbeddings: boolean = true;
```